certs/client.crt

```diff
@@ -0,0 +1,25 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Not sure I follow what all these certs are for?

Reply:
Those should not have committed.  Deleting